### PR TITLE
Add terminal_tutorial.ger.md #2898

### PR DESCRIPTION
**Pull request by mrennings**

### CODE REVIEW

**Comment by CandidDeer** [comment]
@Syknapse Please let me know if we can merge this. Thx.

**Comment by TenTraicion** [comment]
#2880

File: translations/terminal_tutorial/terminal_tutorial.ger.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+      - [Schritt 4: Öffne die Datei index.html](#schritt-4-öffne-die-datei-indexhtml)
+      - [Schritt 5: Kopiere die Karten-Vorlage](#schritt-5-kopiere-die-karten-vorlage)
+      - [Schritt 6: Änderungen eintragen](#schritt-6-änderungen-eintragen)
+      - [Step 7: Commit Deine Änderungen](#step-7-commit-deine-änderungen)
+      - [Schritt 8: Push Deine Änderungen zu GitHub](#schritt-8-push-deine-änderungen-zu-github)
+      - [Schritt 9: Erstelle einen PR(Pull Request)](#schritt-9-erstelle-einen-prpull-request)
+      - [Schritt 10: Feier](#schritt-10-feier)
+    - [Nächste Schritte](#nächste-schritte)
+    - [Danke](#danke)
+    - [Top 100 Beitragende](#top-100-beitragende)
```
Comment (TenTraicion) [inline translations/terminal_tutorial/terminal_tutorial.ger.md:61]: why are there too many steps?

File: translations/terminal_tutorial/terminal_tutorial.ger.md [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+| :----: | :----: | :----: | :----: | :----: |
+[Portuguese (Brazilian)](/translations/terminal_tutorial/terminal_tutorial.por-br.md) | [Ukrainian](/translations/terminal_tutorial/terminal_tutorial.ukr.md) |
+
+> Übersetzungen der Projektdokumentation sind gerne gesehen. Siehe [`Translation Guide` (engl.)](/translations/README.md) um beizutragen.
+
+---
+
+### Setup! :)
+
+Info: Wenn Du Dich nicht so gut mit der Kommandozeile auskennst, [gibt es hier ein GUI-Tutorial mit dem GitHub-Desktop](/README.md)
```
Comment (TenTraicion) [inline translations/terminal_tutorial/terminal_tutorial.ger.md:110]: the readme link should forward to german translation of README

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@mrennings, Thanks for your contribution. Kindly make the required changes.

File: translations/terminal_tutorial/terminal_tutorial.ger.md (hunk 1 above)
Comment (mrennings) [reply translations/terminal_tutorial/terminal_tutorial.ger.md]: Sorry for that, was an Extension auto-generating the TOC, should be fixed now.

File: translations/terminal_tutorial/terminal_tutorial.ger.md [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+#### Steps
+
+- [Beitragen](#beitragen)
+- [Schritt 1: Fork dieses Repository](#schritt-1-fork-dieses-repository)
+- [Schritt 2: Clone das Repository](#schritt-2-clone-das-repository)
+- [Schritt 3: Erstelle einen Branch](#schritt-3-erstelle-einen-branch)
+- [Schritt 4: Öffne die Datei index.html](#schritt-4-öffne-die-datei-indexhtml)
+- [Schritt 5: Kopiere die Karten-Vorlage](#schritt-5-kopiere-die-karten-vorlage)
+- [Schritt 6: Änderungen eintragen](#schritt-6-änderungen-eintragen)
+- [Step 7: Commit Deine Änderungen](#step-7-commit-deine-änderungen)
```
Comment (TenTraicion) [inline translations/terminal_tutorial/terminal_tutorial.ger.md:44]:
```suggestion
- [Schritt 7: Commit Deine Änderungen](#schritt-7-commit-deine-änderungen)
```

File: translations/terminal_tutorial/terminal_tutorial.ger.md [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+- **Name:** Schreibe den Ressourcen-Name in das Textfeld `>hier</a>`.
+- Stelle sicher, dass Du **alle Änderungen gespeichert** hast.
+- **Teste Deine Änderungen**. DIES IST WICHTIG! Öffne die HTML-Datei in Deinem Browser (z. B. durch einen Doppelklick) und schau Dir Deine Karte an, wie sie auf der Seite aussehen wird. Die komplette Seite sollte noch gleich aussehen und nichts sollte kaputt sein. Klicke Deine Links und stell sicher, dass sie funktionieren. Öffne die Konsole (Strg + Shift + J (Windows / Linux) or Cmd + Opt + J (Mac)) und prüfe, dass dort keine Fehlermeldungen sind.
+- Großartig, Du hast die Bearbeitung des Quelltextes abgeschlossen. Als Nächstes wirst Du Deine Änderungen zu GitHub übertragen und sie dann beim Hauptprojekt einreichen
+
+[↑ Nach oben ↑](#schnellzugriff)
+
+---
+
+#### Step 7: Commit Deine Änderungen
```
Comment (TenTraicion) [inline translations/terminal_tutorial/terminal_tutorial.ger.md:261]:
```suggestion
#### Schritt 7: Commit Deine Änderungen
```

**Review by TenTraicion — APPROVED** [review]
just need a simple change

**Comment by TenTraicion** [comment]
@mrennings, Thanks for the contribution.

**Pull request merged**